velox/common/base/RawVector.h

```diff
@@ -152,6 +154,9 @@ class raw_vector {
   }
 
  private:
+  // Constructs  a copy of 'other'. See operator=. 'data_' must be copied.
+  raw_vector(const raw_vector<T>& other) {}
```

Comment:
We can use `raw_vector(const raw_vector<T>& other) = deleted`
Also delete the copy assignment `void operator=(const raw_vector<T>& other) = deleted`